omnisend/src/connection/components/connection-steps.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from '@wordpress/element';
+import { Button, Flex, FlexItem, TextControl } from '@wordpress/components';
+
+const ConnectionSteps = ({ onSubmit }) => {
+	const [apiKey, setApiKey] = useState(null);
+
+	const navigateToExternalUrl = (url) => {
+		window.open(url, '_blank').focus();
+	};
+
+	return (
+		<>
+			<div className="omnisend-spacing-mv-8">
```

Comment:
why are we not using lib spacer?

Reply:
because it is under experimental components:
https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-spacer--docs